proco/schools/loaders/ingest.py

```diff
@@ -67,6 +67,9 @@ def save_data(country, loaded: Iterable[Dict]) -> List[str]:
 
         try:
             school_data['geopoint'] = Point(x=float(data['lon']), y=float(data['lat']))
+
+            if school_data['geopoint'] == Point(x=0, y=0):
+                raise ValueError()
```

Comment:
please return user readable text instead of validation error
```
errors.append(_('Row {0}: Bad data provided for geopoint: zero point').format(row_index))
continue
```